src/main/java/com/github/checkstyle/regression/git/DiffParser.java

```diff
@@ -57,25 +57,36 @@ private DiffParser() {
      * @throws IOException     JGit library exception
      * @throws GitAPIException JGit library exception
      */
+    @SuppressWarnings("TryFinallyCanBeTryWithResources")
```

Comment:
It seems this is coming from IntelliJ and as in main repository, we have it disabled. See https://github.com/checkstyle/checkstyle/blob/7ba18e5138cdb800243b0a7dd15de3bb3741e395/config/intellij-idea-inspections.xml#L2140 .
Please confirm you use the editor IntelliJ and I will copy the `intellij-idea-inspections` from their to regression-tool.